core/src/main/java/tc/oc/pgm/rotation/vote/Formula.java

```diff
@@ -0,0 +1,62 @@
+package tc.oc.pgm.rotation.vote;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import java.util.function.ToDoubleFunction;
+import net.objecthunter.exp4j.Expression;
+import net.objecthunter.exp4j.ExpressionBuilder;
+import net.objecthunter.exp4j.function.Function;
+import org.bukkit.configuration.InvalidConfigurationException;
+
+/** Implementation of different formulas configurable & composable via config sections. */
+public interface Formula extends ToDoubleFunction<Formula.Context> {
```

Comment:
Could we move this to `util` and make it a `final class`?

Reply:
If we were to move it to util, it'd have to be not as part of specifically map voting but a more generic wrapper for Expression from exp4j